### DISCUSSION TITLE
Add FormKitNode findUpwards() function

### DISCUSSION THREAD

**Discussion by mlmoravek** [formkit/formkit#1196] [Ideas]
## Feature: 

A `FormKitNode` element has the [`at()`](https://formkit.com/api-reference/formkit-core#at), [`find()`](https://formkit.com/api-reference/formkit-core#find) and the [`walk()`](https://formkit.com/api-reference/formkit-core#walk) functions to walk through the subtree and find a particular node. For going upwards and searching there are only the [`parent`](https://formkit.com/api-reference/formkit-core#parent) and [`root`](https://formkit.com/api-reference/formkit-core#root) properties. 
However, I'm missing a function to walk up the tree until a node matches some condition.

I would like to have some function like `findUpward()` to find a node which is located up the tree of the current node.

**Comment by justin-schroeder** [comment]
Love the idea.
